src/helpers/data.js

```diff
@@ -42,6 +43,12 @@ var DataHelper = {
         _.each(entries, function (entry, index) {
             entry.__id = index;
 
+            if (entry.time && !entry.date) {
```

Comment:
Adding `date` here means that it can be used as a mapping for an axis. However, in this case it will be mapped as categorical values (since it will be a string), instead of number values (as timestamp is) leading to incorrect display of time proportions. That is caused by the fact that plotting categorical values tries to even out spacing between factors, while numerical values are plotted according to their value.

A more proper solution would be to pass the timestamp through moment in the HUD - that box that shows you values of selected points. There, you can pass a field that is named `ts` or `time` or `timestamp` through moment, and output *that* value to the user, instead of just a timestamp.